### ISSUE TITLE
A Filter for the next rounds in the Matrix

### ISSUE THREAD

**Issue by FrankV-Cachly** [cachly/issues-tracker#5085]
It would be nice to have a filter to find the caches nearby, to fill the next round of the D/T-Matrix.

Find next Round? 
Which round? e.g. 49
Which Type of Caches? e.g. all, Letterboxes, etc.

Thank you in advance,
Frank

**Comment by cachly** [comment]
There is currently not API endpoint for this, as it really would be needed to get all of the grid info. I have put in a request to HQ to see if they would be willing to add it.

**Comment by FrankV-Cachly** [comment]
Hi, with project-gc it is actually possible. So it should exist in the API?

https://project-gc.com/Tools/MapMatrix

Frank

**Comment by cachly** [comment]
No. Project-GC has a special agreement with Geocaching HQ and they have a duplicate database (from what I hear) so they can do much much more than partner apps.

**Comment by FrankV-Cachly** [comment]
Ah, ok. I din't knew this.
